### PR TITLE
Added small client with tests for communicating and interfacing Lubyanka

### PR DESCRIPTION
**Pull request by amanelis**
@JohnDzialo please look over carefully. Thanks.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -54,5 +54,8 @@ lint:
 test:
 	. venv/bin/activate; py.test tests -r w --disable-pytest-warnings
 
+tox:
```
Comment (JohnDzialo) [inline Makefile]: what up with tox?

File: cybric/lubyanka/lubyanka_util.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+class LubyankaUtil(object):
+    def __init__(self, config):
+        self.config = config
+
+        if not hasattr(self.config, 'lubyanka_host'):
+            raise InvalidConfig('missing lubyanka_host, must be an attribute on config passed')
+
+    def create_stash(self, customer_id, stash):
+        return requests.post(urljoin(self.config.lubyanka_host, '/v1/secret-stashes/'), data={
```
Comment (JohnDzialo) [inline cybric/lubyanka/lubyanka_util.py]: why the syncronous call here?

File: cybric/lubyanka/lubyanka_util.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            "customerId": customer_id,
+            "stash": stash
+        })
+
+    async def create_stash_async(self, customer_id, stash):
+        status, result = None, None
+
+        try:
+            url = urljoin(self.config.lubyanka_host, '/v1/secret-stashes/')
+            dat = {"customerId": customer_id, "stash": stash}
```
Comment (JohnDzialo) [inline cybric/lubyanka/lubyanka_util.py]: typo?  data 🤷‍♂️

File: tox.ini [hunk 4]
```diff
@@ -0,0 +1,8 @@
+[tox]
+envlist = py36
+skip_missing_interpreters = True
+
+[testenv]
+changedir = tests
+deps = -rrequirements.txt
```
Comment (JohnDzialo) [inline tox.ini]: missing space between -r and requirements.txt

File: cybric/lubyanka/lubyanka_util.py (hunk 2 above)
Comment (amanelis) [inline cybric/lubyanka/lubyanka_util.py]: I have both, async and normal.

File: Makefile (hunk 1 above)
Comment (amanelis) [inline Makefile]: cause it's dope 😎

File: cybric/lubyanka/lubyanka_util.py (hunk 3 above)
Comment (amanelis) [inline cybric/lubyanka/lubyanka_util.py]: fixing...

**Comment by JohnDzialo** [comment]
so I see you are creating both sync and async calls, for bootstrapping variables or for in process async. 
 i love it!

File: tox.ini (hunk 4 above)
Comment (amanelis) [inline tox.ini]: thanks!

File: cybric/lubyanka/lubyanka_util.py [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+        except AssertionError:
+            self.config.logger.error(f'Exception in assertion of get_stash_async: {status}')
+            raise
+        else:
+            self.config.logger.info(f'Stash fetched: {status}, {result}')
+
+        return status, result
+
+    def update_stash(self, etag, key, stash):
+        return requests.put(urljoin(self.config.lubyanka_host, f'/v1/secret-stashes/{key}'), data=stash, headers={'etag': etag})
```
Comment (JohnDzialo) [inline cybric/lubyanka/lubyanka_util.py]: the async calls return json.  the sync calls just return the raw response.  should we make them both consistent?  I propose our services to always get the json back for all calls.  IMO

**Pull request closed without merging**